src/NServiceBus.AcceptanceTests/NServiceBus.AcceptanceTests.csproj

```diff
@@ -18,9 +18,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" PrivateAssets="All" />
     <PackageReference Include="NUnit" Version="[3.7.1, 4.0.0)" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" PrivateAssets="All" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.9" PrivateAssets="All" />
```

Comment:
as 3.8 wasn't working properly with r#, this needs to some double-checking before merging